src/materials/ShaderMaterial.js

```diff
@@ -34,6 +34,8 @@ function ShaderMaterial( parameters ) {
 	this.vertexShader = 'void main() {\n\tgl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );\n}';
 	this.fragmentShader = 'void main() {\n\tgl_FragColor = vec4( 1.0, 0.0, 0.0, 1.0 );\n}';
 
+	this.isGLSL3 = false;
```

Comment:
Is adding this property good?

Reply:
Another option is we let users write #version in user shader code for GLSL 3 like

    #version 300 es

    out highp vec4 glFragColor;
    void main() {
        glFragColor = vec4( 1.0 );
    }

and we determine GLSL 3 if the user shader code begins with "#version 300 es".